docs/api/boligsameie.md

```diff
@@ -47,18 +47,32 @@ Legg inn eksempel
-| MVA-005  | 403             | Feil i forbindelse med autorisering.                       |
-| MVA-006  | 400             | Feil i forbindelse med validering av inputdata.            |
-| MVA-007  | 404             | Ingen mva-melding funnet på oppgitt referanse.             |
-| MVA-008  | 406             | Feil tilknyttet dataformat. Kun json eller xml er støttet. |
+| Feilkode | HTTP Statuskode | Feilområde                                   |
+|----------|-----------------|----------------------------------------------|
+| GLD_005  | 403             | Feil i forbindelse med autorisering          |
+| GLD_006  | 400             | Feil i request                               |
+| GLD_008  | 400             | Strukturell feil i tilknyttet dataformat     |
+| GLD_010  | 400             | Feil i forbindelse med validering av payload |
+| GLD_011  | 400             | Feil i metadata                              |
+| GLD_017  | 500             | Uspesifisert systemfeil                      |
+| GLD_019  | 409             | Idempotensnøkkel er benyttet tidligere       |
+| GLD_021  | 404             | Finner ikke forespurt ressurs                |
+| GLD_022  | 405             | HTTP-metode ikke støttet                     |
+
+Feilresponsene kan også innholde en feilspesifiseringskode som presiserer feilen ytterligere. 
+Tabellen under viser hvilke feilspesifiseringskoder applikasjonen kan gi.
+Dersom det finnes mer detaljert feilinformasjon enn generelt feilområde vil det beskrives i melding, sti og angitt verdi feltene.
+
+| Feilspesifiseringskode | Feilområde                                     | Årsak                                                                                                                       |
+|------------------------|------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| GLD_1022               | Feil i parametre                               | Diverse feil med parametre i request. Mer detaljert beskrivelse ligger i melding, sti og angitt verdi dersom det er aktuelt |
+| GLD_1023               | Finner ingen ressurs for denne urlen           | Det er ikke noe innhold tilgjengelig på denn URLen                                                                          |
+| GLD_1027               | Inntektsår er ikke støttet                     | Det er ikke tillatt å levere på oppgitt inntektsår                                                                          |
+| GLD_1028               | Header mangler                                 | Påkrevd header er ikke med i requesten                                                                                      |
+| GLD_1030               | Accept-header må være av type application/json | APIet har kun støtte for json responser og accept header må stemme med det                                                  |
```

Comment:
tenker "og content-type header må stemme med det" er unødvendig. Foreslår "Header content-type er feil. APIet har kun støtte for json i request body"

Reply:
Enig, skal fikse